frontend/typescript-examples/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Please, remove folder typescript-examples, only code related to Budgetify should be in PR